Compiling errors

<!-- Welcome to the issues section if it's your first time! -->

### Before creating an issue, please ensure:
- [x] This is a bug in the software that resides in this repository, and not a
      support matter (use https://otland.net/forums/support.16/ for support)
- [x] This issue is reproducible without changes to the C++ code in this repository

### Steps to reproduce (include any configuration/script required to reproduce)

```
Build started...
1>------ Build started: Project: theforgottenserver, Configuration: Release x64 ------
1>otpch.cpp
1>actions.cpp
1>ban.cpp
1>baseevents.cpp
1>bed.cpp
1>chat.cpp
1>combat.cpp
1>condition.cpp
1>configmanager.cpp
1>connection.cpp
1>container.cpp
1>creature.cpp
1>creatureevent.cpp
1>cylinder.cpp
1>database.cpp
1>databasemanager.cpp
1>databasetasks.cpp
1>depotchest.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\chat.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\chat.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\chat.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\chat.cpp)
1>depotlocker.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\condition.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\condition.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\condition.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\condition.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\container.cpp)
1>events.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\bed.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\bed.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\bed.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\creature.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\creature.cpp)
1>C:\Games\forgottenserver\src\combat.h(12,7): message : see declaration of 'Creature' (compiling source file ..\src\creature.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\bed.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\creature.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\configmanager.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\configmanager.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\configmanager.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\configmanager.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\combat.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\combat.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\combat.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\combat.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\actions.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\actions.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\actions.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\actions.cpp)
1>fileloader.cpp
1>C:\Games\forgottenserver\src\iomap.h(113,21): error C2664: 'bool Spawns::loadFromXml(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\spawn.h(70,7): message : see declaration of 'Spawns::loadFromXml' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,21): message : while trying to match the argument list '(std::filesystem::path)' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,21): error C2664: 'bool Houses::loadHousesXML(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,39): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,39): message : No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\house.h(249,7): message : see declaration of 'Houses::loadHousesXML' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,21): message : while trying to match the argument list '(std::filesystem::path)' (compiling source file ..\src\container.cpp)
1>game.cpp
1>globalevent.cpp
1>groups.cpp
1>guild.cpp
1>house.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\events.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\events.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\events.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\events.cpp)
1>housetile.cpp
1>inbox.cpp
1>iologindata.cpp
1>iomap.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\game.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\game.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\game.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\game.cpp)
1>iomapserialize.cpp
1>iomarket.cpp
1>C:\Games\forgottenserver\src\housetile.h(34,1): warning C4266: 'void Tile::internalAddThing(Thing *)': no override available for virtual member function from base 'Tile'; function is hidden (compiling source file ..\src\housetile.cpp)
1>C:\Games\forgottenserver\src\tile.h(226,7): message : see declaration of 'Tile::internalAddThing' (compiling source file ..\src\housetile.cpp)
1>C:\Games\forgottenserver\src\tile.h(116,7): message : see declaration of 'Tile' (compiling source file ..\src\housetile.cpp)
1>item.cpp
1>items.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\guild.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\guild.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\guild.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\guild.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\house.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\house.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\house.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\house.cpp)
1>luascript.cpp
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\housetile.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,21): error C2664: 'bool Spawns::loadFromXml(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\spawn.h(70,7): message : see declaration of 'Spawns::loadFromXml' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,21): message : while trying to match the argument list '(std::filesystem::path)' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,21): error C2664: 'bool Houses::loadHousesXML(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,39): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,39): message : No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\house.h(249,7): message : see declaration of 'Houses::loadHousesXML' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,21): message : while trying to match the argument list '(std::filesystem::path)' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\housetile.h(34,1): warning C4266: 'void Tile::internalAddThing(Thing *)': no override available for virtual member function from base 'Tile'; function is hidden (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\tile.h(226,7): message : see declaration of 'Tile::internalAddThing' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\tile.h(116,7): message : see declaration of 'Tile' (compiling source file ..\src\iomap.cpp)
1>C:\Games\forgottenserver\src\iomap.cpp(57,21): error C2440: 'initializing': cannot convert from 'initializer list' to 'OTB::Loader'
1>C:\Games\forgottenserver\src\iomap.cpp(57,21): message : No constructor could take the source type, or constructor overload resolution was ambiguous
1>mailbox.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\iologindata.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\iologindata.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\iologindata.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\iologindata.cpp)
1>map.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\iomapserialize.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\iomapserialize.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\iomapserialize.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\iomapserialize.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\iomarket.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\iomarket.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\iomarket.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\iomarket.cpp)
1>C:\Games\forgottenserver\src\movement.h(125,1): warning C4242: 'argument': conversion from 'int32_t' to 'unsigned short', possible loss of data (compiling source file ..\src\items.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\item.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\item.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\item.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\item.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\luascript.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\luascript.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\luascript.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\luascript.cpp)
1>monster.cpp
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1735,42): warning C4996: 'fmt::v9::detail::arg_mapper<Context>::map': was declared deprecated
1>        with
1>        [
1>            Context=fmt::v9::format_context
1>        ] (compiling source file ..\src\iomapserialize.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1777,1): message : see reference to function template instantiation 'fmt::v9::detail::value<Context> fmt::v9::detail::make_value<Context,AccessList_t&>(T)' being compiled
1>        with
1>        [
1>            Context=fmt::v9::format_context,
1>            T=AccessList_t &
1>        ] (compiling source file ..\src\iomapserialize.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1895): message : see reference to function template instantiation 'fmt::v9::detail::value<Context> fmt::v9::detail::make_arg<true,Context,fmt::v9::detail::type::int_type,AccessList_t&,0>(T)' being compiled
1>        with
1>        [
1>            Context=fmt::v9::format_context,
1>            T=AccessList_t &
1>        ] (compiling source file ..\src\iomapserialize.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1917): message : see reference to function template instantiation 'fmt::v9::format_arg_store<fmt::v9::format_context,unsigned int,AccessList_t,std::basic_string<char,std::char_traits<char>,std::allocator<char>>>::format_arg_store<uint32_t&,AccessList_t&,std::string&>(uint32_t &,AccessList_t &,std::string &)' being compiled (compiling source file ..\src\iomapserialize.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1918,3): message : see reference to function template instantiation 'fmt::v9::format_arg_store<fmt::v9::format_context,unsigned int,AccessList_t,std::basic_string<char,std::char_traits<char>,std::allocator<char>>>::format_arg_store<uint32_t&,AccessList_t&,std::string&>(uint32_t &,AccessList_t &,std::string &)' being compiled (compiling source file ..\src\iomapserialize.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(3205): message : see reference to function template instantiation 'fmt::v9::format_arg_store<fmt::v9::format_context,unsigned int,AccessList_t,std::basic_string<char,std::char_traits<char>,std::allocator<char>>> fmt::v9::make_format_args<fmt::v9::format_context,uint32_t&,AccessList_t&,std::string&>(uint32_t &,AccessList_t &,std::string &)' being compiled (compiling source file ..\src\iomapserialize.cpp)
1>C:\Games\forgottenserver\src\iomapserialize.cpp(321): message : see reference to function template instantiation 'std::string fmt::v9::format<uint32_t,AccessList_t,std::string>(fmt::v9::basic_format_string<char,unsigned int,AccessList_t,std::basic_string<char,std::char_traits<char>,std::allocator<char>>>,uint32_t &&,AccessList_t &&,std::string &&)' being compiled
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\mailbox.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\mailbox.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\mailbox.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\mailbox.cpp)
1>monsters.cpp
1>mounts.cpp
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1735,42): warning C4996: 'fmt::v9::detail::arg_mapper<Context>::map': was declared deprecated
1>        with
1>        [
1>            Context=fmt::v9::format_context
1>        ] (compiling source file ..\src\iomarket.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1777,1): message : see reference to function template instantiation 'fmt::v9::detail::value<Context> fmt::v9::detail::make_value<Context,MarketAction_t&>(T)' being compiled
1>        with
1>        [
1>            Context=fmt::v9::format_context,
1>            T=MarketAction_t &
1>        ] (compiling source file ..\src\iomarket.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1895): message : see reference to function template instantiation 'fmt::v9::detail::value<Context> fmt::v9::detail::make_arg<true,Context,fmt::v9::detail::type::int_type,MarketAction_t&,0>(T)' being compiled
1>        with
1>        [
1>            Context=fmt::v9::format_context,
1>            T=MarketAction_t &
1>        ] (compiling source file ..\src\iomarket.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1917): message : see reference to function template instantiation 'fmt::v9::format_arg_store<fmt::v9::format_context,MarketAction_t,unsigned short>::format_arg_store<MarketAction_t&,uint16_t&>(MarketAction_t &,uint16_t &)' being compiled (compiling source file ..\src\iomarket.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(1918,3): message : see reference to function template instantiation 'fmt::v9::format_arg_store<fmt::v9::format_context,MarketAction_t,unsigned short>::format_arg_store<MarketAction_t&,uint16_t&>(MarketAction_t &,uint16_t &)' being compiled (compiling source file ..\src\iomarket.cpp)
1>C:\vcpkg\installed\x64-windows\include\fmt\core.h(3205): message : see reference to function template instantiation 'fmt::v9::format_arg_store<fmt::v9::format_context,MarketAction_t,unsigned short> fmt::v9::make_format_args<fmt::v9::format_context,MarketAction_t&,uint16_t&>(MarketAction_t &,uint16_t &)' being compiled (compiling source file ..\src\iomarket.cpp)
1>C:\Games\forgottenserver\src\iomarket.cpp(24,20): message : see reference to function template instantiation 'std::string fmt::v9::format<MarketAction_t&,uint16_t&>(fmt::v9::basic_format_string<char,MarketAction_t &,uint16_t &>,MarketAction_t &,uint16_t &)' being compiled
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\map.cpp)
1>movement.cpp
1>C:\Games\forgottenserver\src\iomap.h(113,21): error C2664: 'bool Spawns::loadFromXml(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\spawn.h(70,7): message : see declaration of 'Spawns::loadFromXml' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,21): message : while trying to match the argument list '(std::filesystem::path)' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,21): error C2664: 'bool Houses::loadHousesXML(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,39): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,39): message : No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\house.h(249,7): message : see declaration of 'Houses::loadHousesXML' (compiling source file ..\src\map.cpp)
1>C:\Games\forgottenserver\src\iomap.h(128,21): message : while trying to match the argument list '(std::filesystem::path)' (compiling source file ..\src\map.cpp)
1>networkmessage.cpp
1>npc.cpp
1>otserv.cpp
1>outfit.cpp
1>outputmessage.cpp
1>party.cpp
1>C:\Games\forgottenserver\src\monster.h(216,1): warning C4266: 'void Creature::onWalk(Direction &)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\monster.cpp)
1>C:\Games\forgottenserver\src\creature.h(180,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\monster.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\monster.cpp)
1>player.cpp
1>C:\Games\forgottenserver\src\movement.h(125,1): warning C4242: 'argument': conversion from 'int32_t' to 'unsigned short', possible loss of data (compiling source file ..\src\movement.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\monster.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\monsters.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\monsters.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\monsters.cpp)
1>podium.cpp
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\monsters.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\movement.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\movement.cpp)
1>C:\Games\forgottenserver\src\combat.h(12,7): message : see declaration of 'Creature' (compiling source file ..\src\movement.cpp)
1>position.cpp
1>protocol.cpp
1>protocolgame.cpp
1>protocollogin.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\npc.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\npc.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\npc.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\npc.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\otserv.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\otserv.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\otserv.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\otserv.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\party.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\party.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\party.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\party.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\player.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\player.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\player.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\player.cpp)
1>protocolold.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\podium.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\podium.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\podium.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\podium.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\protocollogin.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\protocollogin.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\protocollogin.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\protocollogin.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\protocolgame.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\protocolgame.cpp)
1>C:\Games\forgottenserver\src\condition.h(9,7): message : see declaration of 'Creature' (compiling source file ..\src\protocolgame.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\protocolgame.cpp)
1>quests.cpp
1>raids.cpp
1>rsa.cpp
1>scheduler.cpp
1>script.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\protocolold.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\protocolold.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\protocolold.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\protocolold.cpp)
1>scriptmanager.cpp
1>server.cpp
1>signals.cpp
1>spawn.cpp
1>spells.cpp
1>storeinbox.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\quests.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\quests.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\quests.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\quests.cpp)
1>protocolstatus.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\raids.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\raids.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\raids.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\raids.cpp)
1>talkaction.cpp
1>C:\Games\forgottenserver\src\movement.h(125,1): warning C4242: 'argument': conversion from 'int32_t' to 'unsigned short', possible loss of data (compiling source file ..\src\scriptmanager.cpp)
1>tasks.cpp
1>teleport.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\signals.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\signals.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\signals.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\signals.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\spawn.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\spawn.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\spawn.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\spawn.cpp)
1>C:\Games\forgottenserver\src\spells.h(141,1): warning C4242: 'argument': conversion from 'int32_t' to 'unsigned short', possible loss of data (compiling source file ..\src\spells.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\spells.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\spells.cpp)
1>C:\Games\forgottenserver\src\combat.h(12,7): message : see declaration of 'Creature' (compiling source file ..\src\spells.cpp)
1>thing.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\protocolstatus.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\protocolstatus.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\protocolstatus.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\protocolstatus.cpp)
1>tile.cpp
1>tools.cpp
1>trashholder.cpp
1>C:\Games\forgottenserver\src\tools.cpp(165,32): warning C4242: '=': conversion from 'uint32_t' to 'uint8_t', possible loss of data
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\talkaction.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\talkaction.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\talkaction.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\talkaction.cpp)
1>vocation.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\tasks.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\tasks.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\tasks.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\tasks.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\teleport.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\teleport.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\teleport.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\teleport.cpp)
1>weapons.cpp
1>wildcardtree.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\tile.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\tile.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\tile.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\tile.cpp)
1>xtea.cpp
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\trashholder.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\trashholder.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\trashholder.cpp)
1>C:\Games\forgottenserver\src\tile.cpp(1140,30): warning C4245: 'argument': conversion from 'int' to 'uint16_t', signed/unsigned mismatch
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\trashholder.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\vocation.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\vocation.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\vocation.cpp)
1>C:\Games\forgottenserver\src\player.h(453): warning C4242: 'argument': conversion from 'const int32_t' to 'const _Ty', possible loss of data
1>        with
1>        [
1>            _Ty=uint16_t
1>        ] (compiling source file ..\src\vocation.cpp)
1>C:\Games\forgottenserver\src\weapons.h(113,1): warning C4242: 'argument': conversion from 'int32_t' to 'unsigned short', possible loss of data (compiling source file ..\src\weapons.cpp)
1>C:\Games\forgottenserver\src\player.h(1350,1): warning C4266: 'void Creature::onWalk(void)': no override available for virtual member function from base 'Creature'; function is hidden (compiling source file ..\src\weapons.cpp)
1>C:\Games\forgottenserver\src\creature.h(276,15): message : see declaration of 'Creature::onWalk' (compiling source file ..\src\weapons.cpp)
1>C:\Games\forgottenserver\src\creature.h(78,7): message : see declaration of 'Creature' (compiling source file ..\src\weapons.cpp)
1>Done building project "theforgottenserver.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
========== Build started at 11:47 PM and took 19,338 seconds ==========
```

### Expected behaviour
<!-- Tell us what should happen --> Compiling TFS 1.4

### Actual behaviour
<!-- Tell us what happens instead -->

### Environment
<!-- If the issue is environment specific (e.g. compiling errors), include
     name and version of the operating system and compiler you are using. --> x64bit Windows 21H2 (Comp. system 19044.2604) - compiling with Visual Studio 2022  

### Reproduction environment
<!-- If there are specific scripts required to reproduce the issue, provide a link to
     a branch in your fork that has minimum necessary changes to reproduce the issue. -->

1>C:\Games\forgottenserver\src\iomap.h(113,21): error C2664: 'bool Spawns::loadFromXml(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\spawn.h(70,7): message : see declaration of 'Spawns::loadFromXml' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,21): message : while trying to match the argument list '(std::filesystem::path)' (compiling source file ..\src\container.cpp)

have error, yes

yes bro, have a error, and i used a branch 4062 and dont compile too


```
1>C:\Games\forgottenserver\src\iomap.h(113,21): error C2664: 'bool Spawns::loadFromXml(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\container.cpp)
1>C:\Games\forgottenserver\src\iomap.h(113,37): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\container.cpp)
```

> yes bro, have a error, and i used a branch 4062 and dont compile too
> 
> ```
> 1>C:\Games\forgottenserver\src\iomap.h(113,21): error C2664: 'bool Spawns::loadFromXml(const std::string &)': cannot convert argument 1 from 'std::filesystem::path' to 'const std::string &' (compiling source file ..\src\container.cpp)
> 1>C:\Games\forgottenserver\src\iomap.h(113,37): message : Reason: cannot convert from 'std::filesystem::path' to 'const std::string' (compiling source file ..\src\container.cpp)
> ```

yeah, that is why the PR is still in draft, its not completed yet 😄

--- a/src/iomap.cpp
+++ b/src/iomap.cpp
@@ -54,7 +54,7 @@ bool IOMap::loadMap(Map* map, const std::filesystem::path& fileName)
 {
 	int64_t start = OTSYS_TIME();
 	try {
-		OTB::Loader loader{fileName, OTB::Identifier{{'O', 'T', 'B', 'M'}}};
+		OTB::Loader loader{fileName.string(), OTB::Identifier{{'O', 'T', 'B', 'M'}}};
 		auto& root = loader.parseTree();
 
 		PropStream propStream;

--- a/src/iomap.h
+++ b/src/iomap.h
@@ -110,7 +110,7 @@ public:
 			map->spawnfile += "-spawn.xml";
 		}
 
-		return map->spawns.loadFromXml(map->spawnfile);
+		return map->spawns.loadFromXml(map->spawnfile.string());
 	}
 
 	/* Load the houses (not house tile-data)
@@ -125,7 +125,7 @@ public:
 			map->housefile += "-house.xml";
 		}
 
-		return map->houses.loadHousesXML(map->housefile);
+		return map->houses.loadHousesXML(map->housefile.string());
 	}
 
 	const std::string& getLastErrorString() const { return errorString; }

